packages/ember-application/lib/system/application.js

```diff
@@ -1039,7 +1021,6 @@ if (isEnabled('ember-application-visit')) {
       });
     }
   });
```

Comment:
Indentation

(Also, we can just merge this into the actual class definition now instead of reopening the class here.)